src/peakrdl_markdown/exporter.py

```diff
@@ -95,10 +107,12 @@ def _addrnode_header(self, node: AddressableNode, heading_level: int) -> str:
             node_type_name = "address map"
         elif isinstance(node, RegfileNode):
             node_type_name = "register file"
+        elif isinstance(node, MemNode):
+            node_type_name = "memory"
         elif isinstance(node, RegNode):
             node_type_name = "register"
         else:
-            node_type_name = "addressable node"
+            raise ValueError(f"Unknown AddressableNode type {type(node)}")
```

Comment:
Why did you change it to hard fail here? I'd prefer more "graceful" fail, like warning on console and ignoring the field in the output (like it was before with MemNode).